### PR TITLE
Feat/add upload functionality

### PR DESCRIPTION
**Pull request by Labham-Jain**

### CODE REVIEW

File: src/api/routes/global/uploads.ts [hunk 1]
```diff
@@ -15,9 +15,13 @@ export const uploadFile = (router: Router, { s3Client }: { s3Client: S3Client })
...
       }
     }
+    const ext = req.query.ext as string
+    if (!ext) return res.status(422).send('Missing file extension!');
+
+
     const id = req.user.customer_id || req.user.userId;
 
-    const key = `${id}/${ulid()}`
+    const key = `${id}_${ulid()}.${ext}`
```
Comment (ShivamJoker) [inline src/api/routes/global/uploads.ts:24]: why did you change it to `_`, it wont go to sub folder.

Comment (Labham-Jain) [reply src/api/routes/global/uploads.ts:24]: I thought it can be problematic for URLs

Comment (Labham-Jain) [reply src/api/routes/global/uploads.ts:24]: I'll revert
